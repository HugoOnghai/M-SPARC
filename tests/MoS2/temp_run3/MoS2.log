
                            < M A T L A B (R) >
                  Copyright 1984-2019 The MathWorks, Inc.
                  R2019a (9.6.0.1072779) 64-bit (glnxa64)
                               March 8, 2019

 
To get started, type doc.
For product information, visit www.mathworks.com.
 

 Reading .inpt file ...


<INPT>
# $ cat MoS2.inpt
# nprocs: 48
# Test: MoS2 #
LATVEC:
-0.5 0.8666 0
0 0 1
1 0 0
CELL: 6.1 30 6.1
MESH_SPACING: 0.20
FD_ORDER: 12
BC: P D P
KPOINT_GRID: 3 1 3
EXCHANGE_CORRELATION: GGA_PBE
ELEC_TEMP_TYPE: fermi-dirac
ELEC_TEMP: 315.773
TOL_SCF: 1e-6
#ELEC_TEMP: 
CALC_STRESS: 1
PRINT_FORCES: 1
PRINT_ATOMS: 0
MIXING_VARIABLE: potential
MIXING_PRECOND: none

# MD
#MD_FLAG:  1
#MD_METHOD: NVT_NH
#MD_TIMESTEP: 1
#MD_NSTEP: 10
#PRINT_RESTART_FQ: 3
#RESTART_FLAG: 0

<\INPT>

 Reading .ion file ...
 Number of atom types : 2
 Total number of atoms: 3
 Default atomic mass for Mo is 95.950000
 Default atomic mass for S is 32.067500
 pseudo_dir # 1 = Mo.psp8
 pseudo_dir # 2 = S.psp8

 COORD:
   0.070149695000000   3.296864329500000  15.000000000000000
   2.806003050000000   1.762084904600000  18.655220000000000
   3.050003050000000   1.762084904600000  12.344780000000000

   5.403031739800300

   0.031202687423158

 kpoint grid before symmetry:
  -0.343343459408721                   0  -0.343343459408721
                   0                   0  -0.343343459408721
   0.343343459408721                   0  -0.343343459408721
  -0.343343459408721                   0                   0
                   0                   0                   0
   0.343343459408721                   0                   0
  -0.343343459408721                   0   0.343343459408721
                   0                   0   0.343343459408721
   0.343343459408721                   0   0.343343459408721

 kpoint grid after symmetry:
                   0                   0                   0
   0.343343459408721                   0                   0
  -0.343343459408721                   0   0.343343459408721
                   0                   0   0.343343459408721
   0.343343459408721                   0   0.343343459408721

## Chebyshev polynomial degree not provided, finding npl ...
## Based on the mesh size, npl is set to: 35
## Number of states not provided, finding Nev ...
## Based on the number of electrons, Nev is set to: 21
## Poisson tolerance not provided, choosing poisson_tol ...
## poisson_tol is set to: 1.000e-08
## Pseudocharge tolerance not provided, choosing pseudocharge_tol ...
## pseudocharge_tol is set to: 1.000e-08
 Creating differentiation matrices ...
 Done. (5.751 sec)

 Estimated memory usage:
 Total:    1.22 GB
 orbitals            :  870.94 MB
 sparse matrices     :  350.45 MB
 global-size vectors :   14.52 MB
 mixing histories    :   14.52 MB

 Finding rb for Mo ...
 rb = {2.847 2.847 2.847}, int_b = -14.000107859548018, err_rb = 1.079e-04
 rb = {3.050 3.050 3.050}, int_b = -14.000106116110375, err_rb = 1.061e-04
 rb = {3.253 3.253 3.253}, int_b = -14.000131249719336, err_rb = 1.312e-04
 rb = {3.457 3.457 3.457}, int_b = -14.000095516061752, err_rb = 9.552e-05
 rb = {3.660 3.660 3.660}, int_b = -14.000083817557417, err_rb = 8.382e-05
 rb = {3.863 3.863 3.863}, int_b = -14.000069869440821, err_rb = 6.987e-05
 rb = {4.067 4.067 4.067}, int_b = -14.000059491302949, err_rb = 5.949e-05
 rb = {4.270 4.270 4.270}, int_b = -14.000052154526045, err_rb = 5.215e-05
 rb = {4.473 4.473 4.473}, int_b = -14.000047442620843, err_rb = 4.744e-05
 rb = {4.677 4.677 4.677}, int_b = -14.000043593232055, err_rb = 4.359e-05
 rb = {4.880 4.880 4.880}, int_b = -14.000041396357959, err_rb = 4.140e-05
 rb = {5.083 5.083 5.083}, int_b = -14.000036833091082, err_rb = 3.683e-05
 rb = {5.287 5.287 5.287}, int_b = -14.000032829272223, err_rb = 3.283e-05
 rb = {5.490 5.490 5.490}, int_b = -14.000029074686337, err_rb = 2.907e-05
 rb = {5.693 5.693 5.693}, int_b = -14.000028177802673, err_rb = 2.818e-05
 rb = {5.897 5.897 5.897}, int_b = -14.000024139731789, err_rb = 2.414e-05
 rb = {6.100 6.100 6.100}, int_b = -14.000018233490390, err_rb = 1.823e-05
 rb = {6.303 6.303 6.303}, int_b = -14.000016169113199, err_rb = 1.617e-05
 rb = {6.507 6.507 6.507}, int_b = -14.000014665721309, err_rb = 1.467e-05
 rb = {6.710 6.710 6.710}, int_b = -14.000013146255320, err_rb = 1.315e-05
 rb = {6.913 6.913 6.913}, int_b = -14.000014001172579, err_rb = 1.400e-05
 rb = {7.117 7.117 7.117}, int_b = -13.999998243881693, err_rb = 1.756e-06
 rb = {7.320 7.320 7.320}, int_b = -14.000000370321091, err_rb = 3.703e-07
 rb = {7.523 7.523 7.523}, int_b = -13.999999927834264, err_rb = 7.217e-08
 rb = {7.727 7.727 7.727}, int_b = -14.000000009946369, err_rb = 9.946e-09
 rb = {7.727 7.727 7.727}
 Finding rb for S ...
 rb = {2.440 2.440 2.440}, int_b = -5.999383434374065, err_rb = 6.166e-04
 rb = {2.643 2.643 2.643}, int_b = -5.999772222124866, err_rb = 2.278e-04
 rb = {2.847 2.847 2.847}, int_b = -5.999946952436908, err_rb = 5.305e-05
 rb = {3.050 3.050 3.050}, int_b = -6.000008812291928, err_rb = 8.812e-06
 rb = {3.253 3.253 3.253}, int_b = -6.000028706824276, err_rb = 2.871e-05
 rb = {3.457 3.457 3.457}, int_b = -6.000031642032393, err_rb = 3.164e-05
 rb = {3.660 3.660 3.660}, int_b = -6.000029569051671, err_rb = 2.957e-05
 rb = {3.863 3.863 3.863}, int_b = -6.000025373944013, err_rb = 2.537e-05
 rb = {4.067 4.067 4.067}, int_b = -6.000021072252857, err_rb = 2.107e-05
 rb = {4.270 4.270 4.270}, int_b = -6.000018028382751, err_rb = 1.803e-05
 rb = {4.473 4.473 4.473}, int_b = -6.000016480752660, err_rb = 1.648e-05
 rb = {4.677 4.677 4.677}, int_b = -6.000015507209617, err_rb = 1.551e-05
 rb = {4.880 4.880 4.880}, int_b = -6.000015677560459, err_rb = 1.568e-05
 rb = {5.083 5.083 5.083}, int_b = -6.000014106067416, err_rb = 1.411e-05
 rb = {5.287 5.287 5.287}, int_b = -6.000013008440804, err_rb = 1.301e-05
 rb = {5.490 5.490 5.490}, int_b = -6.000012139650024, err_rb = 1.214e-05
 rb = {5.693 5.693 5.693}, int_b = -6.000013362995103, err_rb = 1.336e-05
 rb = {5.897 5.897 5.897}, int_b = -6.000012190326669, err_rb = 1.219e-05
 rb = {6.100 6.100 6.100}, int_b = -6.000009339941091, err_rb = 9.340e-06
 rb = {6.303 6.303 6.303}, int_b = -6.000009090619258, err_rb = 9.091e-06
 rb = {6.507 6.507 6.507}, int_b = -6.000009063815733, err_rb = 9.064e-06
 rb = {6.710 6.710 6.710}, int_b = -6.000008802885315, err_rb = 8.803e-06
 rb = {6.913 6.913 6.913}, int_b = -6.000010104707597, err_rb = 1.010e-05
 rb = {7.117 7.117 7.117}, int_b = -5.999998728444387, err_rb = 1.272e-06
 rb = {7.320 7.320 7.320}, int_b = -6.000000268218047, err_rb = 2.682e-07
 rb = {7.523 7.523 7.523}, int_b = -5.999999947878253, err_rb = 5.212e-08
 rb = {7.727 7.727 7.727}, int_b = -6.000000007159445, err_rb = 7.159e-09
 rb = {7.727 7.727 7.727}

 ###############################################################
 Relaxation step number: 1 

 Starting pseudocharge generation and self energy calculation...
 Integration b = 26.000000011500

 Integration b_ref = 25.999999999955

 ****************************************
 *          Eself_ref = 478.738373       *
 ****************************************
 Done. (3.941420 s)
 Time for b calculation: 3.955 seconds.

 Starting calculating nonlocal projectors ...
 Done. (1.838606 s)

 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ 
 Starting SCF iteration...
 AAR converged to a relative residual of 8.72253e-09 in 277 iterations.
 Poisson problem took 26.877539s
 ============================================= 
 Relaxation iteration:  1
 SCF iteration number:  1, Chebyshev cycle: 1 
 ============================================= 
 Fermi energy = 4.537526
 This SCF iteration took 185.006 s.

 ============================================= 
 Relaxation iteration:  1
 SCF iteration number:  1, Chebyshev cycle: 2 
 ============================================= 
 Fermi energy = 0.336406
 This SCF iteration took 157.577 s.

 ============================================= 
 Relaxation iteration:  1
 SCF iteration number:  1, Chebyshev cycle: 3 
 ============================================= 
 Fermi energy = -0.138824
 This SCF iteration took 156.242 s.

 ============================================= 
 Relaxation iteration:  1
 SCF iteration number:  1, Chebyshev cycle: 4 
 ============================================= 
 Fermi energy = -0.199855
 AAR converged to a relative residual of 5.36992e-09 in 271 iterations.
 Poisson problem took 26.254570s
 Escc = 1.06120533
 Etot = -89.00397190
 Eatom = -29.66799063
 Error in SCF iteration: 1.5417e-01 
 This SCF iteration took 183.962 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number:  2 
 ========================= 
 Fermi energy = -0.194532
 AAR converged to a relative residual of 8.90387e-09 in 301 iterations.
 Poisson problem took 29.083659s
 Escc = -1.51729564
 Etot = -88.88867576
 Eatom = -29.62955859
 Error in SCF iteration: 4.7165e-01 
 This SCF iteration took 186.887 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number:  3 
 ========================= 
 Fermi energy = -0.202287
 AAR converged to a relative residual of 9.13583e-09 in 313 iterations.
 Poisson problem took 30.119993s
 Escc = 1.21967684
 Etot = -89.08075919
 Eatom = -29.69358640
 Error in SCF iteration: 5.1821e-02 
 This SCF iteration took 184.783 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number:  4 
 ========================= 
 Fermi energy = -0.193040
 AAR converged to a relative residual of 8.9199e-09 in 247 iterations.
 Poisson problem took 23.753197s
 Escc = 0.11507522
 Etot = -89.08259699
 Eatom = -29.69419900
 Error in SCF iteration: 3.0513e-02 
 This SCF iteration took 178.694 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number:  5 
 ========================= 
 Fermi energy = -0.191371
 AAR converged to a relative residual of 9.20659e-09 in 259 iterations.
 Poisson problem took 24.745328s
 Escc = 1.04552276
 Etot = -89.08144712
 Eatom = -29.69381571
 Error in SCF iteration: 4.2918e-02 
 This SCF iteration took 182.824 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number:  6 
 ========================= 
 Fermi energy = -0.183568
 AAR converged to a relative residual of 7.24995e-09 in 271 iterations.
 Poisson problem took 25.935076s
 Escc = -0.33604719
 Etot = -89.08350256
 Eatom = -29.69450085
 Error in SCF iteration: 1.9818e-02 
 This SCF iteration took 182.083 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number:  7 
 ========================= 
 Fermi energy = -0.183501
 AAR converged to a relative residual of 7.0618e-09 in 247 iterations.
 Poisson problem took 23.864123s
 Escc = 0.02353549
 Etot = -89.08392381
 Eatom = -29.69464127
 Error in SCF iteration: 7.9313e-03 
 This SCF iteration took 180.822 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number:  8 
 ========================= 
 Fermi energy = -0.183736
 AAR converged to a relative residual of 8.59094e-09 in 193 iterations.
 Poisson problem took 18.548272s
 Escc = -0.00616393
 Etot = -89.08396789
 Eatom = -29.69465596
 Error in SCF iteration: 1.4336e-03 
 This SCF iteration took 176.286 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number:  9 
 ========================= 
 Fermi energy = -0.183935
 AAR converged to a relative residual of 9.6607e-09 in 139 iterations.
 Poisson problem took 13.451467s
 Escc = 0.00038191
 Etot = -89.08396859
 Eatom = -29.69465620
 Error in SCF iteration: 3.0543e-04 
 This SCF iteration took 170.959 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number: 10 
 ========================= 
 Fermi energy = -0.184029
 AAR converged to a relative residual of 7.59924e-09 in 163 iterations.
 Poisson problem took 15.716584s
 Escc = -0.00066365
 Etot = -89.08396927
 Eatom = -29.69465642
 Error in SCF iteration: 1.6159e-04 
 This SCF iteration took 172.088 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number: 11 
 ========================= 
 Fermi energy = -0.184044
 AAR converged to a relative residual of 8.56262e-09 in 91 iterations.
 Poisson problem took 8.777022s
 Escc = -0.00009165
 Etot = -89.08396923
 Eatom = -29.69465641
 Error in SCF iteration: 2.7364e-05 
 This SCF iteration took 164.450 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number: 12 
 ========================= 
 Fermi energy = -0.184053
 AAR converged to a relative residual of 7.52609e-09 in 67 iterations.
 Poisson problem took 6.543613s
 Escc = 0.00009805
 Etot = -89.08396900
 Eatom = -29.69465633
 Error in SCF iteration: 1.5188e-05 
 This SCF iteration took 163.077 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number: 13 
 ========================= 
 Fermi energy = -0.184051
 AAR converged to a relative residual of 7.5878e-09 in 67 iterations.
 Poisson problem took 6.539615s
 Escc = 0.00017443
 Etot = -89.08396917
 Eatom = -29.69465639
 Error in SCF iteration: 4.0981e-06 
 This SCF iteration took 161.753 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number: 14 
 ========================= 
 Fermi energy = -0.184049
 AAR converged to a relative residual of 8.93488e-09 in 31 iterations.
 Poisson problem took 3.074262s
 Escc = 0.00011138
 Etot = -89.08396935
 Eatom = -29.69465645
 Error in SCF iteration: 3.0942e-06 
 This SCF iteration took 159.074 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number: 15 
 ========================= 
 Fermi energy = -0.184048
 AAR converged to a relative residual of 8.70061e-09 in 31 iterations.
 Poisson problem took 3.034060s
 Escc = 0.00006827
 Etot = -89.08396928
 Eatom = -29.69465643
 Error in SCF iteration: 2.9578e-06 
 This SCF iteration took 155.305 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number: 16 
 ========================= 
 Fermi energy = -0.184046
 AAR converged to a relative residual of 8.79996e-09 in 25 iterations.
 Poisson problem took 2.483571s
 Escc = 0.00001086
 Etot = -89.08396958
 Eatom = -29.69465653
 Error in SCF iteration: 2.4357e-06 
 This SCF iteration took 153.536 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number: 17 
 ========================= 
 Fermi energy = -0.184047
 AAR converged to a relative residual of 8.76477e-09 in 43 iterations.
 Poisson problem took 4.234138s
 Escc = -0.00001490
 Etot = -89.08396915
 Eatom = -29.69465638
 Error in SCF iteration: 1.1274e-06 
 This SCF iteration took 159.174 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number: 18 
 ========================= 
 Fermi energy = -0.184046
 AAR converged to a relative residual of 9.34653e-09 in 31 iterations.
 Poisson problem took 3.077207s
 Escc = -0.00002161
 Etot = -89.08396927
 Eatom = -29.69465642
 Error in SCF iteration: 8.1143e-07 
 This SCF iteration took 157.577 s.


 Finished SCF iteration in 18 steps!
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ 

 **********************************************************
 *          Energy per unit cell = -89.083969271 Ha.       *
 *          Energy per atom = -29.694656424 Ha.            *
 **********************************************************

 Starting atomic force calculation ... 
 local force calculation: 3.812 s
 ***********************************************************
 *                      Atomic Force                       *
 ***********************************************************
 Drift free forces (Ha/Bohr):
  -0.023508218467431   0.038942966641180   0.107269473839070
   0.012523138395969  -0.011088729191348  -0.068722060603971
   0.010985080071461  -0.027854237449832  -0.038547413235099

 Max magnitude of forces (Ha/Bohr):   0.116515797229558


 Time for calculating forces: 8.944355 s.

["Stress in Ha/Bohr**2"


]  -0.000627712912223  -0.000515319024904
  -0.000515319024904   0.004626441386321


["Stress in GPa (equivalent to all periodic system)"


]  -0.615598381447409  -0.505373637346485
  -0.505373637346485   4.537153488196694


 Time for calculating stress: 26.017756 s.

 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ 
 Final atomic positions (Cartesian) are as follows:
  3.806260 	 15.000000 	  1.972333 
  2.034343 	 18.655220 	  3.822669 
  2.034343 	 12.344780 	  4.066669 

 Run-time of the program: 3649.723663 seconds

ans = 

  struct with fields:

                  cell_typ: 2
                   lat_vec: [3x3 double]
                  metric_T: [3x3 double]
                    grad_T: [3x3 double]
                    lapc_T: [3x3 double]
                      Jacb: 0.866168945680640
                        L1: 6.100000000000000
                        L2: 30
                        L3: 6.100000000000000
                        Nx: 30
                        Ny: 151
                        Nz: 30
                         N: 135900
                      ecut: -1
              mesh_spacing: 0.200000000000000
                   kptgrid: [5x3 double]
                  kptshift: [0 0 0]
                      nkpt: [3 1 3]
                     tnkpt: 5
                      wkpt: [5x1 double]
                        BC: 3
                       BCx: 0
                       BCy: 1
                       BCz: 0
                      isBS: 0
                   lattice: 'undefined'
                   SCF_tol: 1.000000000000000e-06
                       Nev: 21
               poisson_tol: 1.000000000000000e-08
          pseudocharge_tol: 1.000000000000000e-08
                       Cst: 27.211386245987999
              Ha_Bohr3_GPa: 2.942101569650548e+04
                        kB: 3.166811563455600e-06
               elec_T_type: 0
                      Temp: 3.157730000000000e+02
                       bet: 1.000006412214051e+03
                       npl: 35
                       FDn: 6
                    rc_ref: 0.500000000000000
              max_relax_it: 100
                dbg_switch: 0
                        xc: 2
                 Nelectron: 26
                     n_typ: 2
                TimeRevSym: 1
                  spin_typ: 0
                     nspin: 1
              CheFSI_Optmz: 0
          chefsibound_flag: 0
               FixRandSeed: 0
                rhoTrigger: 4
                 NetCharge: 0
                 MAXIT_SCF: 100
                 MINIT_SCF: 3
             MAXIT_POISSON: 1000
            accuracy_level: -1
     target_force_accuracy: -1
    target_energy_accuracy: -1
                 TOL_RELAX: 5.000000000000000e-04
               TOL_LANCZOS: 0.010000000000000
               precond_tol: 4.088636571742514e-05
               precond_kTF: 1
            precond_thresh: 0.250000000000000
            precond_fitpow: 2
          precond_resta_q0: 1.360000000000000
          precond_resta_Rs: 2.760000000000000
            MixingVariable: 1
             MixingPrecond: 0
                  Pf_guess: []
             MixingHistory: 7
           MixingParameter: 0.300000000000000
     MixingParameterSimple: 0.300000000000000
            PulayFrequency: 1
          PulayRestartFreq: 0
                    TWtime: 1.000000000000000e+09
                 RelaxFlag: 0
                 RelaxMeth: 'LBFGS'
            max_dilatation: 1.200000000000000
            TOL_RELAX_CELL: 5.000000000000000e-06
                    MDFlag: 0
               RestartFlag: 0
                    MDMeth: 'NVE'
                     MD_dt: 1
                  MD_Nstep: 0
                     ion_T: -1
                thermos_TF: -1
              ion_elec_eqT: 1
              ion_vel_dstr: 2
                NLCG_sigma: 0.500000000000000
                     qmass: 1
                 L_history: 20
               L_finit_stp: 0.005000000000000
                  L_maxmov: 0.200000000000000
               L_autoscale: 1
                 L_lineopt: 1
                   L_icurv: 1
                   FIRE_dt: 1
                 FIRE_mass: 1
               FIRE_maxmov: 0.200000000000000
               Calc_stress: 1
                 Calc_pres: 0
            PrintForceFlag: 1
          PrintAtomPosFlag: 0
          PrintAtomVelFlag: 1
         PrintElecDensFlag: 0
            PrintEigenFlag: 0
                PrintMDout: 1
             PrintRelaxout: 1
              Printrestart: 1
           Printrestart_fq: 1
                       xin: 0
                       yin: 0
                       zin: 0
                     twist: 0
                  filename: 'MoS2'
            inputfile_path: '.'
                        XC: 'GGA_PBE'
                       Atm: [1x2 struct]
                     n_atm: 3
                     Atoms: [3x3 double]
           mvAtmConstraint: [3x3 double]
                    IsFrac: [2x1 double]
                    IsSpin: [2x1 double]
            is_default_psd: 0
                  temp_tol: 1.000000000000000e-12
                 xc_rhotol: 1.000000000000000e-14
                Atoms_init: [3x3 double]
                  lat_uvec: [3x3 double]
                        dx: 0.203333333333333
                        dy: 0.200000000000000
                        dz: 0.203333333333333
                        dV: 0.007162254770839
                        w2: [1x7 double]
                        w1: [1x7 double]
                         W: [135900x1 double]
                    occfac: 2
                Relax_iter: 1
              elecgs_Count: 1
               StressCount: 1
                    amu2au: 1.822888485332371e+03
                    fs2atu: 41.341373336493000
                 isOutl_11: [390x1 logical]
                 isOutr_11: [390x1 logical]
                      I_11: [390x1 double]
                     II_11: [390x1 double]
                      V_11: [390x1 double]
                      I_22: [1921x1 double]
                     II_22: [1921x1 double]
                      V_22: [1921x1 double]
                 isOutl_33: [390x1 logical]
                 isOutr_33: [390x1 logical]
                      I_33: [390x1 double]
                     II_33: [390x1 double]
                      V_33: [390x1 double]
                  isOutl_1: [360x1 logical]
                  isOutr_1: [360x1 logical]
                       I_1: [360x1 double]
                      II_1: [360x1 double]
                       V_1: [360x1 double]
                       I_2: [1770x1 double]
                      II_2: [1770x1 double]
                       V_2: [1770x1 double]
                  isOutl_3: [360x1 logical]
                  isOutr_3: [360x1 logical]
                       I_3: [360x1 double]
                      II_3: [360x1 double]
                       V_3: [360x1 double]
                 G_JOutl_1: [1630800x1 logical]
                 G_JOutr_1: [1630800x1 logical]
                     G_J_1: [1630800x1 double]
                     G_I_1: [1630800x1 double]
                     G_V_1: [1630800x1 double]
                     G_J_2: [1593000x1 double]
                     G_I_2: [1593000x1 double]
                     G_V_2: [1593000x1 double]
                 G_JOutl_3: [1630800x1 logical]
                 G_JOutr_3: [1630800x1 logical]
                     G_J_3: [1630800x1 double]
                     G_I_3: [1630800x1 double]
                     G_V_3: [1630800x1 double]
                   Lap_std: [135900x135900 double]
                    grad_1: [135900x135900 double]
                    grad_2: [135900x135900 double]
                    grad_3: [135900x135900 double]
                LapPreconL: [135900x135900 double]
                LapPreconU: [135900x135900 double]
              memory_usage: 1.311163208000000e+09
               staticfname: 'MoS2.static'
                 suffixNum: 0
                  outfname: 'MoS2.out'
                  parallel: 0
                         b: [135900x1 double]
                     b_ref: [135900x1 double]
                    rho_at: [135900x1 double]
                     Eself: 2.119678144491131e+02
                 Eself_ref: 4.787383725487983e+02
                       V_c: [135900x1 double]
                 PosCharge: 26.000000011499793
                 NegCharge: -26.000000011499793
                    E_corr: -2.507319514961637e-04
                       rho: [135900x1 double]
                      Atom: [1x3 struct]
                       phi: [135900x1 double]
                      e_xc: [135900x1 double]
                 dvxcdgrho: [135900x1 double]
                       Vxc: [135900x1 double]
                      Veff: [135900x1 double]
                         X: [135900x7 double]
                         F: [135900x7 double]
          mixing_hist_fkm1: [135900x1 double]
          mixing_hist_xkm1: [135900x1 double]
                       psi: [135900x21x5 double]
    upper_bound_guess_vecs: [135900x5 double]
                    EigVal: [21x5 double]
                  lambda_f: -0.184046320268381
                       occ: [21x5 double]
                    Etotal: -89.083969270672313
                     Eband: -21.030613432970497
                       Exc: -12.270350728859775
                    Exc_dc: -15.980813061551672
                  Eelec_dc: 1.402042686173326e+02
                      Eent: -2.603770871218214e-10
                      Escc: -2.160640139705351e-05
                   S_Debug: [1x1 struct]
                     force: [3x3 double]
                 abs_force: [3x1 double]
                    Stress: [2x2 double]
             stress_dimred: [2x3 double]

