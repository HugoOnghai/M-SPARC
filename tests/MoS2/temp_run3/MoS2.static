Atomic forces (Ha/Bohr):
     -0.0235082185       0.0389429666       0.1072694738
      0.0125231384      -0.0110887292      -0.0687220606
      0.0109850801      -0.0278542374      -0.0385474132
Stress (Ha/Bohr**2)                :
 -6.2771291222E-04  -5.1531902490E-04 
 -5.1531902490E-04   4.6264413863E-03 
Stress equiv. all periodic (GPa)   :
 -6.1559838145E-01  -5.0537363735E-01 
 -5.0537363735E-01   4.5371534882E+00 
