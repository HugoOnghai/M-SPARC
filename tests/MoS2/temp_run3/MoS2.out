***************************************************************************
*                          M-SPARC (Feb 20, 2020)                         *
*   Copyright (c) 2019 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                Date: 11-May-2021  Start time: 21:08:46                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 6.100000 30.000000 6.100000 
LATVEC:
-0.500000000000000 0.866600000000000 0.000000000000000 
0.000000000000000 0.000000000000000 1.000000000000000 
1.000000000000000 0.000000000000000 0.000000000000000 
FD_GRID: 30 150 30
FD_ORDER: 12
BC: P D P
KPOINT_GRID: 3 1 3
KPOINT_SHIFT: 0 0 0
ELEC_TEMP_TYPE: fermi-dirac
SMEARING: 0.000999994
CHEB_DEGREE: 35
NSTATES: 21
EXCHANGE_CORRELATION: GGA_PBE
CALC_STRESS: 1
MAXIT_SCF: 100
TOL_SCF: 1.00E-06
TOL_POISSON: 1.00E-08
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-08
MIXING_VARIABLE: potential
MIXING_PRECOND: none
MIXING_PARAMETER: 0.30
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART_FREQ: 0
REFERENCE_CUTOFF: 0.50
RHO_TRIGGER: 4
FIX_RAND: 0
PRINT_FORCES: 1
PRINT_ATOMS: 0
PRINT_EIGEN: 0
PRINT_DENSITY: 0
OUTPUT_FILE: MoS2.out
***************************************************************************
                             Initialization                                
***************************************************************************
Mesh spacing in x-direction        :  0.203333 (Bohr)
Mesh spacing in y-direction        :  0.200000 (Bohr)
Mesh spacing in z direction        :  0.203333 (Bohr)
Number of symmetry adapted k-points:  5
Output printed to                  :  MoS2.out
Total number of atom types         :  2
Total number of atoms              :  3
Total number of electrons          :  26
Atom type 1  (valence electrons)   :  Mo 14
Pseudopotential                    :  Mo.psp8
lloc                               :  4
Pseudocharge radii of atom type 1  :  7.73 7.73 7.73
Number of atoms of type 1          :  1
Atom type 2  (valence electrons)   :  S 6
Pseudopotential                    :  S.psp8
lloc                               :  4
Pseudocharge radii of atom type 2  :  7.73 7.73 7.73
Number of atoms of type 2          :  2
Estimated total memory usage       :  1.22 GB
=====================================================================
                    Self Consistent Field (SCF#1)                     
=====================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -2.9667990635E+01        1.542E-01        682.785
2            -2.9629558587E+01        4.717E-01        186.885
3            -2.9693586395E+01        5.182E-02        184.780
4            -2.9694198996E+01        3.051E-02        178.691
5            -2.9693815706E+01        4.292E-02        182.821
6            -2.9694500855E+01        1.982E-02        182.080
7            -2.9694641271E+01        7.931E-03        180.821
8            -2.9694655963E+01        1.434E-03        176.284
9            -2.9694656196E+01        3.054E-04        170.957
10           -2.9694656422E+01        1.616E-04        172.086
11           -2.9694656409E+01        2.736E-05        164.448
12           -2.9694656334E+01        1.519E-05        163.072
13           -2.9694656390E+01        4.098E-06        161.750
14           -2.9694656449E+01        3.094E-06        159.072
15           -2.9694656427E+01        2.958E-06        155.303
16           -2.9694656526E+01        2.436E-06        153.534
17           -2.9694656384E+01        1.127E-06        159.172
18           -2.9694656424E+01        8.114E-07        157.575
Total number of SCF: 18    
====================================================================
                                Energy                              
====================================================================
Free energy per atom               : -2.9694656424E+01 (Ha/atom)
Total free energy                  : -8.9083969271E+01 (Ha)
Band structure energy              : -2.1030613433E+01 (Ha)
Exchange correlation energy        : -1.2270350729E+01 (Ha)
Self and correction energy         : -2.1196806518E+02 (Ha)
Entropy*kb*T                       : -2.6037708712E-10 (Ha)
Fermi level                        : -1.8404632027E-01 (Ha)
Average force                      :  7.8684801474E-02 (Ha/Bohr)
Maximum force                      :  1.1651579723E-01 (Ha/Bohr)
Time for force calculation         :  8.944 (sec)
Max stress                         :  4.6264413863E-03 (Ha/Bohr**2)
Max stress equiv. all periodic     :  4.5371534882E+00 (GPa)
Time for stress calculation        :  26.020 (sec)
***************************************************************************
                               Timing info                                 
***************************************************************************
Total walltime                     :  3649.724 sec
___________________________________________________________________________

***************************************************************************
*             Material Physics & Mechanics Group, Georgia Tech            *
*                       PI: Phanish Suryanarayana                         *
*                Main Developers: Qimen Xu, Abhiraj Sharma                *
*     Acknowledgements: U.S. DOE (DE-SC0019410); NSF (1333500,1553212)    *
***************************************************************************
                                                                           
